frontend/src/Routes.js

```diff
@@ -9,15 +9,15 @@ import { UserProfilePage } from 'src/pages/UserProfilePage';
 export const route = {
   home: () => `/`,
   verification: () => "/verification/",
-  userProfile: () => `/user-profile`
+  userProfile: (username) => `/profile/${username}`,
```

Comment:
nevím jestli je vhodné tady mít username, vzhledem k tomu, že to není povinný atribut a nevyplňuje se při registraci, tzn po registraci uživatele ještě není vyplněný...
mohli bychom použít id?

Reply:
Co kdyby byl povinný a vyplňoval se při registraci? V každé aplikaci mám username, ale ve skoro žádné nemám first name a last name, alespoň dokud neřešíme kreditky a platby.

Reply:
no ja jsem ten registrační formulář nenavrhoval, tohle by měl rozhodnout asi PO